ttnn/cpp/ttnn/device.cpp

```diff
@@ -26,6 +26,58 @@ void disable_and_clear_program_cache(Device &device) {
     device.disable_and_clear_program_cache();
 }
 
+float sfpu_positive_nan(DataType dtype) {
+    switch (dtype) {
+        case DataType::BFLOAT16: {
+            return float(0x7FFF) ;
+        }
+        case DataType::FLOAT32: {
+            return float(0x7FFF) ;
```

Comment:
this seems to be incorrect, base on the https://github.com/tenstorrent/tt-metal/blob/main/tech_reports/Handling_Special_Value/special_values.md.
Positive NaN for fp32 should be `0x7FFF_FFFF`

Reply:
Addressed